pkg/apis/application/v1alpha1/app_types.go

```diff
@@ -65,6 +65,10 @@ type AppSpec struct {
 	// +nullable
 	// Config is the config to be applied when the app is deployed.
 	Config AppSpecConfig `json:"config,omitempty"`
+	// +kubebuilder:validation:Optional
+	// +nullable
+	// Install is the config used to deploy the app and is passed to chart operator.
```

Comment:
```suggestion
	// Install is the config used when installing the app.
```

chart-operator is an implementation detail and may even change to the Flux helm-controller in the future. So can we drop that part?